### PR TITLE
Handle single string=>string flags without quotes

### PR DESCRIPTION
**Pull request by tamalsaha**
```
OK: --f1 "a=5,6" --f2 b=3,4 --f3 "c=5,6",d=7
Not OK: --f4 c="5,6"
```

### CODE REVIEW

**Comment by tamalsaha** [comment]
@eparis , this pr adds support for `--f2 b=3,4` flag.

cc: @sgraham785

xref: #133 #129

**Comment by tamalsaha** [comment]
@eparis , if you can take a look at this, that will be great.

File: string_to_string_test.go [hunk 1]
```diff
@@ -140,13 +140,14 @@ func TestS2SCalledTwice(t *testing.T) {
 	var s2s map[string]string
 	f := setUpS2SFlagSet(&s2s)
 
-	in := []string{"a=1,b=2", "b=3", `"e=5,6"`, `f="7,8"`}
```
Comment (eparis) [inline string_to_string_test.go]: Can you add a check that `len(s2s) == len(expected)` so we can't make this mistake again?

Comment (tamalsaha) [reply string_to_string_test.go]: Added this check. PTAL

**Pull request merged**